Hello @sea-bass,

I have added an example to test the viewer. The example is running fine but I have the following warning:

```
(viser) Connection opened (0, 1 total), 5 persistent messages
[Entering video recording context]
connection handler failed
Traceback (most recent call last):
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/websockets/asyncio/server.py", line 376, in conn_handler
    await self.handler(connection)
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/viser/infra/_infra.py", line 395, in ws_handler
    await asyncio.gather(
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/viser/infra/_infra.py", line 591, in _message_producer
    "messages": tuple(
                ^^^^^^
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/viser/infra/_infra.py", line 592, in <genexpr>
    message.as_serializable_dict() for message in outgoing
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/viser/infra/_messages.py", line 115, in as_serializable_dict
    k: _prepare_for_serialization(v, hints[k]) for k, v in vars(self).items()
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jvaillan/dev/inria/pinocchio/pinocchio/.pixi/envs/collision/lib/python3.12/site-packages/viser/infra/_messages.py", line 58, in _prepare_for_serialization
    return int(value)
           ^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
(viser) Connection opened (1, 2 total), 298 persistent messages
```

Is it normal ?

--- a/bindings/python/pinocchio/visualize/viser_visualizer.py
+++ b/bindings/python/pinocchio/visualize/viser_visualizer.py
@@ -326,8 +326,8 @@ class ViserVisualizer(BaseVisualizer):
         Capture an image from the Viser viewer and return an RGB array.
 
         Parameters:
-            w: The width of the captured image.
-            h: The height of the captured image.
+            w: The width of the captured image. If None, uses the actual camera width.
+            h: The height of the captured image. If None, uses the actual camera height.
             client_id: The ID of the Viser client handle.
                 If None, uses the first available client.
             transport_format: The transport format to use for the captured image.
@@ -346,7 +346,11 @@ class ViserVisualizer(BaseVisualizer):
         else:
             cli = clients[client_id]
 
-        return cli.get_render(height=h, width=w, transport_format=transport_format)
+        height = h or cli.camera.image_height
+        width = w or cli.camera.image_width
+        return cli.get_render(
+            height=height, width=width, transport_format=transport_format
+        )
 
     def setBackgroundColor(self, preset_name: str = "gray", col_top=None, col_bot=None):
         raise NotImplementedError("setBackgroundColor is not yet implemented.")

